src/main/java/me/karakelley/http/Server.java

```diff
@@ -16,7 +16,7 @@ public Server(int port, ServerLogger logger) {
   public void start() {
     try {
       serverSocket = new ServerSocket(port);
-      logger.info("Started on port " + port);
+      logger.info("Started on port " + serverSocket.getLocalPort());
```

Comment:
Don't hate me but I have two comments on this one line:

1. Aren't you testing logging? Why is there no corresponding test change?
2. This is not the ideal way to interpolate values when using a logging framework.